Sources/Purchasing/Purchases.swift

```diff
@@ -523,6 +523,18 @@ extension Purchases {
         subscriberAttributesManager.setPushToken(pushToken, appUserID: appUserID)
     }
 
+    /**
+     * Subscriber attribute associated with the push token for the user.
+     *
+     * #### Related Articles
+     * -  [Subscriber attributes](https://docs.revenuecat.com/docs/subscriber-attributes)
+     *
+     * - Parameter pushToken: `nil` will delete the subscriber attribute.
```

Comment:
Maybe reference the other method here, and this method in the other one?

Reply:
great idea! adding